active-requirements.md

```diff
@@ -4,6 +4,7 @@ These details describe the requirements to acquire an "active" status each seaso
 
 An Indie must satisfy at least one of the following requirements to be "active" within a season:
 
-- Provide at least 260 billable hours to client projects through the protocol
-- Provide (and document) at least 40 hours of contributions to internal efforts of the DAO
-- Earn at least 2,000 $INDIE during the period of the season
+- Engage in at least 2 billable Indie Protocol sprints for client projects
+- Provide (and document) at least 40 hours of contributions to internal efforts of the DAO, approved and accepted by a Committee Steward
```

Comment:
do you think there could be room to use this as backup if 5,000 INDIE is not reached. just thinking out loud

Reply:
Agreed - so any of these can be achieved in order to be considered active (you don't need to satisfy all three)

Reply:
Yup, I guess I asked it wrongly, I was thinking about a possible risk of stretching hours to stay active cause 5000 requirement wasn't met, but I guess we could fallback to the core approval item, then, depending how it goes, we can adjust it next season if needed.